pbr/lib/uiComponents.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
```

Comment:
If no one is privy to the idea of having a dart file for universal components, then I can easily move this back into donation file